server/controllers/apply/applications/withdrawalsController.ts

```diff
@@ -50,6 +54,14 @@ export default class WithdrawalsController {
     return async (req: Request, res: Response) => {
       try {
         const body: NewWithdrawal = { reason: req.body.reason, otherReason: req.body.otherReason }
+
+        if (body.reason === 'other' && !body.otherReason) {
```

Comment:
Is it worth adding a cheeky `.trim` in here to check that the user doesn't just enter a space (intentionally or by mistake)?